scripting/serverhop.sp

```diff
@@ -475,6 +481,7 @@ public void OnSocketError(Handle sock, const int errorType, const int errorNum,
 	delete g_hSocket[i];
 }
 
-Action timerErrorCooldown(Handle timer) {
+public Action timerErrorCooldown(Handle timer) {
```

Comment:
Timer callbacks don't need 'public' declaraction, but I am going to accept anyways.

Reply:
Should be good now.